cmake/ProjectWasm3.cmake

```diff
@@ -26,6 +26,7 @@ ExternalProject_Add(wasm3
     -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
     -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
     -DCMAKE_BUILD_TYPE=Release
+    -DBUILD_WASI_SUPPORT=FALSE
```

Comment:
Is this actually working ?

Reply:
They added that on my request recently.
This allows building wasm3 on older Linux distributions where `getrandom()` is not available.
And also builds quicker.

Reply:
But we are using my fork 😉

Reply:
I asked you to rebase some time ago. This fixed build on old Ubuntu 16.04 so I don't care more than that.

Reply:
I did the rebase when you asked, but haven't since. Maybe the WASI stuff was included in that rebase, I'm not sure.

Tried rebasing again, but current `master` fails our tests.